### PR TITLE
Docs: fix examples for prefer-numeric-literals

### PR DESCRIPTION
**Pull request by mobilutz**
**What is the purpose of this pull request? (put an "X" next to item)**

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

`Number.parseInt("1F7", 16)` -> `503`

**Is there anything you'd like reviewers to focus on?**

Just check it in the console. 😉

### CODE REVIEW

File: docs/rules/prefer-numeric-literals.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ parseInt("767", 8) === 503;
 parseInt("1F7", 16) === 255;
```
Comment (not-an-aardvark) [inline docs/rules/prefer-numeric-literals.md]: Should this line also be corrected?

**Comment by kaicataldo** [comment]
@mobilutz Friendly ping - anything we can do to help you land this?

**Comment by mobilutz** [comment]
@not-an-aardvark @kaicataldo Sorry, the mail got lost in my inbox ;)
I fixed the second `parseInt`, it should be good to be merged now.

**Comment by platinumazure** [comment]
Can you please sign our CLA? We can't merge anything unless the author signs the CLA. Thanks!

**Comment by mobilutz** [comment]
Done

**Comment by kaicataldo** [comment]
Thank you! Looks like a random CI failure - restarting.

**Comment by kaicataldo** [comment]
Thanks for contributing to ESLint!

**Pull request merged**